baseapp/testapp.go

```diff
@@ -2,8 +2,8 @@ package baseapp
 
 import (
 	abci "github.com/tendermint/abci/types"
-	"github.com/tendermint/go-crypto"
 
+	"github.com/cosmos/cosmos-sdk/baseapp/testtx"
```

Comment:
I don't see the point of moving this out?

Reply:
There was a point when I was rearranging when circular dependencies where created in baseapp when it references auth.. since then auth is not referenced in baseapp anymore - but I figured I'd leave this in just because it was a bit annoying to deal with when I did. Will move back in if you think it's better.... also maybe `auth` should be adjusted to not include reference to testing whether it's this one `testtx` type... is there another way around that? thoughts?